contrib/epee/include/net/net_utils_base.h

```diff
@@ -128,7 +128,7 @@ namespace net_utils
 				case ipv4_network_address::ID:
 					if (!is_store)
 						const_cast<network_address&>(this_ref).reset(new ipv4_network_address(0, 0));
-					KV_SERIALIZE(as<ipv4_network_address>());
+					KV_SERIALIZE(template as<ipv4_network_address>());
```

Comment:
That has nothing to do with what the patch claims, and was already fixed by someone else.